src/style.css

```diff
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
+}
```

Comment:
it's allowed to use only box-sizing: border-box for * selector
If you need to reset default margin you can apply it to the tag selectors